### DISCUSSION TITLE
Only Fans downloads no longer working

### DISCUSSION THREAD

**Discussion by zappycat** [aclap-dev/video-downloadhelper#50] [Help and Q&A] [answered]
### Your question

Only Fans for all older content still downloads but apparently they implemented the option to DRM vids. Is there anything being looked into to erectify those of us hard done by this issue?

### Addon & CoApp version and details

version: 8.1.6.0
beta: false
build date: 2024-01-18
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-GB
license: accepted
key (only first 16 fisrt characters): e8f2b77e37994c6a
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\rossg",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by GrampaWildWilly** [comment]
Don't expect anything.  DRM is specifically intended to block downloading.  If you are desperate for your own copy of such content, you'll have to use a screen recorder.

**Reply by zappycat** [reply]
Thanks, also appreciate your condescending wording assuming 'desperation' of 'such content'. Is viewing adult video content beneath non-adult content? How pathetic people who view such filth must be! The shame! 

I wonder why I would want to save copies of content I've paid to stream? It's more money spent on content than the average use-case of this softwware to rip videos which are free with ad-support. DRM can always be broken, and always has. Even Denuvo, the most successful DRM in history was eventually broken even though it still poses a challenge. I'm interested if the developers are looking at ways to circumvent this.

**Reply by GrampaWildWilly** [reply]
No condescension was implied or expressed.  It's an unfortunate fact that facial expression & tone of voice are not communicated in forums such as these.  Don't be so quick to assume bad manners in others.  As for adult content, don't impute to others attitudes that have not been expressed.  I wasn't even thinking about adult content when I responded.  I try to be civil here.  I hope to set an example that way.

As for DRM, the devs have repeatedly said back on the old Google forum that DRM is not in their plans.

**Comment by paulrouget** [comment, marked as answer]
Sadly we do not support DRM and there's no way around it :/
